packages/landing/src/web/Block/BlockItem.tsx

```diff
@@ -138,12 +138,14 @@ const renderBlockItem = (props) => {
           </Box>
         )
       case BlockItemTypeEnum.IMAGE:
-        return (
-          <Box {...boxProps}>
-            <Image src={title} {...titleProps} />
-          </Box>
-        )
-      case BlockItemTypeEnum.STORAGE_IMAGE:
+        const isAbsolutePath = !/^public.+/.test(title)
+        if (isAbsolutePath) {
```

Comment:
I'd also leave a comment here to say something like:

// If it's absolute path, this is a local src, so don't fetch from network.